docs/release/trg-0/trg-8-01.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 8.01 - KIT General
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 18-Jan-2024  | Initial contribution                   |
+
+## Why
+
+KIT means Keep It Together. A KIT contains all the artefacts that are relevant to participate in the Catena-X data space as an operator, data provider or solution / app provider. These stakeholders have one place with all the information they need. For example an explaination of an API, the specification of it and the deployment instructions. The target of a KIT should be that you have no need to visit any other information source.
```

Comment:
Typo "artifacts" instead of "artefacts",
Typo "explanation" instead of "explaination"

"goal ..." or "aim ..." instead of "target of a KIT"?